pyaedt/circuit.py

```diff
@@ -360,6 +361,25 @@ def create_schematic_from_netlist(self, file_to_import):
         self.logger.info("Netlist correctly imported into %s", self.design_name)
         return True
 
+    @aedt_exception_handler
+    def read_ibis(self, path):
+        """Create an IBIS model based on the data contained into an *.ibs file.
+
+        Parameters
+        ----------
+        path : str
+            Path of the ibis file.
+
+        Returns
+        ----------
+        :class:`pyaedt.generic.ibis_reader.Ibis`
+            Ibis object exposing all data from the ibis file.
+        """
```

Comment:
You may not choose to provide an example within `IbisReader`, but one should be provided here.

Reply:
Excellent idea. Done.